nexus/src/context.rs

```diff
@@ -67,13 +68,13 @@ pub struct ConsoleConfig {
 impl ServerContext {
     /// Create a new context with the given rack id and log.  This creates the
     /// underlying nexus as well.
-    pub fn new(
+    pub async fn new(
```

Comment:
Did this really become `async`? I'm missing the `await` inside the body.

Reply:
It's a remnant from pulling it out of https://github.com/oxidecomputer/omicron/pull/1056 - it'll be necessary to use DNS services to look up where CRDB is running.

I'll remove it for now, however.